src/Products/PloneMeeting/tests/testVotes.py

```
@@ -1194,11 +1194,11 @@ def test_pm_ItemGetVotesResult(self):
         self.assertTrue(may_view_field(item, 'votesResult'))
 
         # get outside meeting
-        self.assertEqual(cfg.getVotesResultTALExpr(), '')
+        self.assertEqual(cfg.votes_result_tal_expr, '')
         self.assertEqual(item.getVotesResult(), '')
         self.assertEqual(item.getVotesResult(real=True), '')
         self.assertFalse(isinstance(item.getVotesResult(), unicode))
```

Ruff (0.15.11)

[error] Undefined name `unicode`

(F821)